src/test/java/org/cactoos/iterable/RangeOfTest.java

```diff
@@ -116,4 +121,30 @@ void testLocalDateRange() {
         ).affirm();
     }
 
+    @Test
+    void shouldBeTraversableMultipleTimes() {
+        final Iterable<Character> range = new RangeOf<>('a', 'c', value -> ++value);
+        final Collection<Character> copy = new ArrayList<>(6);
+        range.forEach(copy::add);
+        range.forEach(copy::add);
+        new Assertion<>(
+            "Must add elements two times",
+            copy,
+            Matchers.iterableWithSize(6)
+        ).affirm();
+    }
+
+    @Test
+    void shouldNotChangeAfterTraversing() {
+        final Iterable<Character> range = new RangeOf<>('a', 'c', value -> ++value);
+        range.forEach(Assertions::assertNotNull);
```

Comment:
@andreoss same comment as before, we should not be using this. Also can you rebase on master? The CI should fail with this now :)

Reply:
@victornoel Sorry, forgot about the second test. Fixed